website/content/docs/operators/deployment/schedulers/mesos-local-mac.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Setting up Heron with Mesos Cluster Locally on Mac
+---
+
+This is a step by step guide to run Heron on a Mesos cluster locally.
+
+## Install Heron
+Follow [Quick Start Guide](../../../../getting-started) to install Heron.
+
+## Setting up an Apache Mesos Cluster Locally
+
+Follow [Installing Mesos on your Mac with Homebrew](https://mesosphere.com/blog/2014/07/07/installing-mesos-on-your-mac-with-homebrew/) to have Mesos installed and running. To confirm Mesos cluster is ready for accepting Heron topologies, make sure you can access Mesos management console [http://localhost:5050](http://localhost:5050) and see activated slaves.
```

Comment:
s/to have Mesos installed and running/to install and run Mesos/

s/make sure you can access/access the/

Also please limit line lenghts here and elsewhere to 100 chars to make commenting and reading in an IDE easier.